src/editor/components/Setting/ComponentAttr/FormElementRenderer.tsx

```
@@ -0,0 +1,107 @@
+
+  // 从静态配置中提取渲染所需的信息
+  const {
+    type,
+    options,
+    component: componentName,
+    props: customSetterProps, // 这是 meta.tsx 中为 custom setter 定义的 props
+  } = setting;
+
+  // 从静态配置中提取应直接传递给 antd 控件的额外 props
+  const {
+    name: _omitName,
```

ESLint

[error] '_omitName' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)